src/smartMirrorProject/theMirror/views.py

```diff
@@ -276,9 +334,17 @@ def index(request):
             # The codes for the stations can be retrieved with the NS API (https://apiportal.ns.nl/api-details#api=reisinformatie-api&operation=getStations). We've saved
             # an csv file with the station codes in Teams (Algemeen\NS_API)
             "data": fetch_reisplanner(
-                "DID", "AH", 1
+                "AH", "DID", 1
             ),  # vanaf station, naar station, aantal journeys die hij laat zien
         },
+        "Traveldepartures": {
+            "id": 3,
+            "type": "traveldepartures",
+            "data": fetch_departures(
+                "AH", destination_filter=["Nijmegen", "Winterswijk", "Doetinchem"]
+            ),  # Filter for multiple destinations
+            "station": "Arnhem Centraal",  # API boodt geen mogelijkheid aan om deze ook aan te roepen helaas
```

Comment:
Comments graag in het engels. 
Ik snap ook niet zo goed wat je bedoeld met dit commentaar en zou dit stukje niet een onderdeel moeten zijn van data